src/ansys/aedt/core/modeler/circuits/primitives_maxwell_circuit.py

```diff
@@ -85,8 +85,8 @@ def create_resistor(self, name=None, value=50, location=None, angle=0, use_insta
 
         Parameters
         ----------
-        name : str, optional
-            Name of the resistor. The default is ``None``.
+        name : str, int, float or optional
+            Name of the resistor. The default is ``None`` which adds a resistor without a name.
```

Comment:
@DaveTwyman are you sure that by default it creates a resistor without a name?

Reply:
@gmalinve , Yes that is the default and intended behavior for all the Maxwell Circuit Editor Components.